policy/modules/admin/fapolicyd.if

```diff
@@ -152,6 +152,8 @@ interface(`fapolicyd_admin',`
 	files_search_runtime($1)
 	admin_pattern($1, fapolicyd_runtime_t)
 
-	fapolicyd_run_fagenrules($1, $2)
+	ifndef(`direct_sysadm_daemon',`
+		fapolicyd_run_fagenrules($1, $2)
+	')
```

Comment:
Please move this to the end of the interface.

Reply:
Also, wouldn't this prevent the admin from generating rules on DIRECT_INITRC systems?

Reply:
I'm not sure.  Is there a better way to solve this issue?  
in /lib/systemd/system/fapolicyd.service has an ExecStartPre that runs /usr/sbin/fagenrules - which is why there is init_script_domain(fagenrules_t, fagenrules_exec_t)
 in fapolicyd.te (and thus the conflict).

Reply:
Ok. I missed that.